src/main/resources/static/api.yml

```diff
@@ -703,6 +704,17 @@ components:
             - admin
             - doctor
             - nurse
+    ErrorResponse:
+      type: object
+      properties:
+        status:
+          type: string
```

Comment:
status should be an integer, shouldn't it?
then you don't need to do `HttpStatus.NOT_FOUND.toString()`, you can just return the status